tests/phpunit/integration/Core/Admin/PointerTest.php

```diff
@@ -134,6 +134,21 @@ public function test_is_active( array $args, $expected ) {
 		}
 	}
 
+	public function test_get_buttons() {
+		$buttons = 'function(event, container) { return jQuery("<button>OK</button>"); }';
```

Comment:
This should be updated since JS can't be passed through from this class any more.